src/jit/gentree.cpp

```diff
@@ -1043,19 +1043,16 @@ int GenTreeCall::GetNonStandardAddedArgCount(Compiler* compiler) const
         // R11 = PInvoke cookie param
         return 1;
     }
-    else if (gtCallType == CT_INDIRECT)
+    else if (IsVirtualStub())
     {
-        if (IsVirtualStub())
-        {
-            // R11 = Virtual stub param
-            return 1;
-        }
-        else if (gtCallCookie != nullptr)
-        {
-            // R10 = PInvoke target param
-            // R11 = PInvoke cookie param
-            return 2;
-        }
+        // R11 = Virtual stub param
```

Comment:
A nit: it bothers me that registers are given in the comments, but I don't know which platform (or platforms) the registers refer to.

Reply:
I would prefer to change comments there in a separate PR.